sp800-63c/sec8_security.md

```diff
@@ -95,7 +95,9 @@ described in the last subsection.
 
 	1.  The assertion is digitally signed by the IdP. The RP 
     checks the digital signature to verify that it was issued by a
-    legitimate IdP.
+    legitimate IdP. The key used is stored in suitably secure storage available to the application 
+	(e.g., keychain storage, trusted platform module, secure element) and is communicated to the RP in 
+	a secure manner (e.g., through a trusted out-of-band mechanism or delivery over a secure protected channel).
```

Comment:
secure protected channel -> authenticated protected channel
because we have defined the latter term and it includes requirements like use of approved crypto and sufficient key lengths. (also in the other sections you changed below)